src/environments/.env.ts

```
@@ -1,9 +1,17 @@
+/**
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The /** key should be with a value or have an equal sign

(KeyWithoutValue)